### PR TITLE
Fixed the Exponent method #13

### PR DESCRIPTION
**Pull request by aimananees**
Make sure these boxes are checked before submitting this pull request~!~ Thank you!!

- [] The test of the method you have fixed is passing
- [] You have written a new skeleton method for someone else to work on!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,11 +6,14 @@
...
-    "jest": "^21.2.1",
-    "eslint": "^4.9.0"
+    "eslint": "^4.9.0",
+    "jest": "^21.2.1"
   },
   "scripts": {
     "test": "jest",
     "lint": "eslint ./lib ./test"
+  },
+  "dependencies": {
```
Comment (BlakeGuilloud) [inline package.json]: If you could run `yarn remove ganon` and push up the resolved lock file + package.json, i would appreciate it!

**Comment by BlakeGuilloud** [comment]
@aimananees You are going to need to pull down the latest master, resolve the merge conflicts, and push it back up. I'm going to remove your other Pull Requests and keep this one open until the conflicts are resolved. Thanks!

**Comment by BlakeGuilloud** [comment]
Hey @aimananees ! We decided on going with double quotes for our style standard if you don't mind fixing this to reflect the current linting rules, we can get this update to exponent in!

File: test/cubeRoot.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { cubeRoot } = require('../lib');
+
+test('cube root of 27 to equal 3', () => {
```
Comment (andreasgruenh) [inline test/cubeRoot.test.js]: We decided on double quotes project wide, please adapt these parts :)

**Comment by ktilcu** [comment]
I'm gonna close this out and open the issue back up to contributors. The new method namespace has been taken and it's been >24 hours since last communication.

**Pull request closed without merging**